certified-connectors/Datablend/apiProperties.json

```diff
@@ -0,0 +1,37 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "h0n3r0rjg0adkoldvu6ulv1br",
```

Comment:
@bmp02050 Please remove the client id value as per Microsoft's security policy. You can replace it with dummy text.
`"clientId": "[[DUMMY]]"`

Reply:
Completed